spec/models/metric/ci_mixin/capture_spec.rb

```diff
@@ -172,149 +159,6 @@ def verify_perf_capture_queue_historical(last_perf_capture_on, total_queue_items
     end
   end
 
-  context "2 collection periods total, end of 1. period has incomplete stat" do
```

Comment:
I'm not very familiar with these tests but this context and the one that follows both seem to be generic and maybe should remain. Again, not sure though.